docs/src/atopics.md

```diff
@@ -0,0 +1,335 @@
+# Advanced Topics
+
+## Live plotting
+
+Plotting while a computational heavy simulation is performed can be easily achieved with FerriteVis.jl.
+Every plotter object of type `MakiePlotter` holds a property called `u` which is a so called `Observable`.
+If an `Observable` changes, all its dependencies are triggered to change as well. So, all we need to do is to update
+the observable `plotter.u`.
+For this purpose the function [`FerriteVis.update!`](@ref) is provided. It takes a `plotter:MakiePlotter`
+and a new solutiuon vector `u_new` and updates `plotter.u`, thereby all open plots called with `plotter` are updated.
+
+A summary of the needed steps for live plotting:
+1. Create a plotter before your time stepping begins
+2. Call a plot or the `ferriteviewer` and save the return in a variable, e.g. `fig`
+3. `display(fig)` in order to force the plot/viewer to pop up, even if its called inside a function body
+4. `FerriteVis.update!(plotter,u_new)` where `u_new` corresponds to your new solution of the time step
+
+As an illustrative example, let's consider the [plasticity example of Ferrite.jl](https://ferrite-fem.github.io/Ferrite.jl/dev/examples/plasticity/).
+In order to have live plotting we only need a few changes in the `solve` function.
+
+```julia
```

Comment:
This has quite some distractions. Can we move some of the code (which is not immediately necessary) into a separate file for now or something?

Reply:
The only thing that I can do is remove everything except for the solving function and refer to the `docs/src/ferrite-examples/plasticity.jl` file for the full code

Reply:
Since this code is not executed anyway maybe we may want to put an include statement or similar here to show the reader the actual point of the example. I think we can assume that in the advanced topics the reader should be familiar with what the called functions do on a conceptual level.

Reply:
checkout the current docs page